src/common/components/select-repository-list/index.js

```diff
@@ -99,10 +101,20 @@ class SelectRepositoryList extends Component {
         }
         { this.renderPageLinks.call(this) }
         <div className={ styles.footer }>
-          <div className={ styles.right }>
-            <Button onClick={ this.onSubmit.bind(this) } appearance={ 'positive' }>
-              Add
-            </Button>
+          <HeadingThree>
+            { selectedRepos.length } selected
+          </HeadingThree>
+          <div className={ styles['footer-right'] }>
+            <div className={ styles['button-wrapper'] }>
+              <LinkButton appearance="neutral" to="/dashboard">
+                Cancel
+              </LinkButton>
```

Comment:
We should probably hide this button in the case where you have no snaps, since otherwise `/dashboard` is going to redirect right back to `/dashboard/select-repositories`.